.flowconfig

```diff
@@ -1,7 +1,7 @@
 [ignore]
 .*/src/api/.*
 .*/src/core/.*
-.*/dist/.*
+.*/(?<!ripple-keypairs/)dist/.*
```

Comment:
I don't think this is working as intended. I tried changing 'ripple-keypairs' to some other string and it still worked. Maybe we should just delete the "dist" line; it still works if you do that. It would be slightly slower because it's checking more files, but at least we wouldn't have this problem again.